app/views/admin/enterprises/_change_type_form.html.haml

```diff
@@ -27,10 +27,10 @@
               %monthly-pricing-description{ joiner: "newline" }
 
           %p.description
-            = t('.producer_description_text')
+            = t('.sell_description_text')
             %br
             %br
-            = t('.producer_description_text2')
+            = t('.sell_description_text2')
```

Comment:
If `producer_description_text` and `producer_description_text2` keys are not used anymore, please remove them from locale yml files.